PsfLauncher/main.cpp

```diff
@@ -304,7 +315,87 @@ void LaunchMonitorInBackground(std::filesystem::path packageRoot, const wchar_t
         }
     }
 }
+
+void LogTelemetryData()
+{
+	auto configRoot = PSFQueryConfigRoot();
+
+	if (auto applications = configRoot->as_object().try_get("applications"))
+	{
+		for (auto& applicationsConfig : applications->as_array())
+		{
+			auto exeStr = applicationsConfig.as_object().try_get("executable")->as_string().wide();
+			auto idStr = applicationsConfig.as_object().try_get("id")->as_string().wide();
+			TraceLoggingWrite(
+				g_Log_ETW_ComponentProvider,
+				"ApplicationsConfigdata",
+				TraceLoggingWideString(exeStr, "applications_executable"),
+				TraceLoggingWideString(idStr, "applications_id"),
+				TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage),
+				TraceLoggingKeyword(MICROSOFT_KEYWORD_CRITICAL_DATA));
+		}
+	}
+
+	if (auto processes = configRoot->as_object().try_get("processes"))
+	{
+		for (auto& processConfig : processes->as_array())
+		{
+			auto exeStr = processConfig.as_object().get("executable").as_string().wide();
+			TraceLoggingWrite(
+				g_Log_ETW_ComponentProvider,
+				"ProcessesExecutableConfigdata",
+				TraceLoggingWideString(exeStr, "processes_executable"),
+				TraceLoggingBoolean(TRUE, "UTCReplace_AppSessionGuid"),
+				TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage),
+				TraceLoggingKeyword(MICROSOFT_KEYWORD_CRITICAL_DATA));
+
+			if (auto fixups = processConfig.as_object().try_get("fixups"))
+			{
+				for (auto& fixupConfig : fixups->as_array())
+				{
+					auto dllStr = fixupConfig.as_object().try_get("dll")->as_string().wide();
+					TraceLoggingWrite(
+						g_Log_ETW_ComponentProvider,
+						"ProcessesFixUpConfigdata",
+						TraceLoggingWideString(dllStr, "processes_fixups"),
+						TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage),

```

Comment:
If we dont know the app that is using these fixups, then this could be productandperformance instead of usage.  No??

Reply:
we're trying to gather usage data and feel to me like it should be PDT_ProductAndServiceUsage and not PDT_ProductAndServicePerformance as we're not trying to gather perfomance